static/app/utils/replays/frame.tsx

```diff
@@ -74,11 +73,13 @@ export function getBreadcrumbType(frame: ReplayFrame): BreadcrumbType {
       case 'issue':
         return BreadcrumbType.ERROR;
       case 'ui.slowClickDetected':
-        return (frame as SlowClickFrame).data.endReason === 'timeout'
+        return isDeadClick(frame as SlowClickFrame)
           ? BreadcrumbType.ERROR
           : BreadcrumbType.WARNING;
       case 'ui.multiClick':
-        return BreadcrumbType.ERROR;
+        return isRageClick(frame as MultiClickFrame)
```

Comment:
This shouldn't occur right now because we only have crumbs that are `isDeadClick` or `isDeadRageClick`, right?

Reply:
that is right. but I thought i should leave in `case 'ui.multiClick':` and make it do the right thing so if the breadcrumb renders, it'll render nice.
weak opinion though, i can take it out to save the code.